src/Umbraco.Web.UI.Client/src/views/propertyeditors/mediapicker3/umbMediaPicker3PropertyEditor.component.js

```diff
@@ -0,0 +1,429 @@
+(function () {
```

Comment:
The file name doesn't follow the convention of the other .js files. All lower case.